Funcky.Test/TestUtils/CheckAssert.cs

```diff
@@ -0,0 +1,29 @@
+using FsCheck;
+
+namespace Funcky.Test.TestUtils;
```

Comment:
Missing newline

Reply:
Fixed in 8641b9fb7912c9aa38f778aaea2464c4c1c44dc0